build/target-repository/bootstrap.php

```diff
@@ -1,28 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-// inspired by https://github.com/phpstan/phpstan/blob/master/bootstrap.php
-
-use Composer\Autoload\ClassLoader;
-
-spl_autoload_register(static function (string $class): void {
-    static $composerAutoloader;
-    // already loaded in bin/ecs.php
-    if (defined('__ECS_RUNNING__')) {
-        return;
-    }
```

Comment:
also this comment is interessting, since this `__ECS_RUNNING__` constant [is defined in the `ecs.php`](https://github.com/easy-coding-standard/easy-coding-standard/blob/133816266d414a55c3af72510b81404a5360aaa7/bin/ecs.php#L17), which effectively makes this autoloader use-less

Reply:
No, bootstrap is loaded in both:

- project vendor/autoload.php
- vendor/easy-coding-standard/bin/ecs.php

when it loaded from `vendor/autoload.php`, it doesn't have `__ECS_RUNNING__` yet, so prefixed vendor need to be loaded, while in `bin/ecs.php`, it load the scoper so prefixed classes exists.

Reply:
the `bootstrap.php` is needed for loading the scoper vendor's prefixed

https://github.com/easy-coding-standard/easy-coding-standard/tree/11.2.10/vendor

classes is used by application is the class in vendor is resolvable. The usage is same in Rector, which we can create:

```php
new \PhpParser\Node\Stmt\Expression($expr);
```

on our custom rector because of the bootstrap.

Reply:
said differently: is there a valid use case to add ECS as a composer dependency and and not use the CLI command?

Reply:
> the `bootstrap.php` is needed for loading the scoper vendor's prefixed

could you describe steps to test this (even manually) so I can see whether my PR breaks ?

Reply:
That's will require rebuild rector without `bootstrap.php`, which will **broke** entire user that uses `dev-main`. 

Here the flow of Rector use case:

Rector is distributed with prefixed vendor https://github.com/rectorphp/rector/tree/0.15.20/vendor which include the `nikic/php-parser` inside it, that need to be loaded as the root composer.json doesn't define it in the require https://github.com/rectorphp/rector/blob/0.15.20/composer.json, so the bootstrap is needed to load the prefixed vendor/autoload.php

Reply:
That's not the case, if `bootstrap.php` removed, it means the classes no longer autoload outside `bin/ecs` usage, that means if user want to use:

```php
new \Symplify\EasyCodingStandard\Application\FileProcessorCollector(...);
```

or any `vendor/` prefixed class for whatever reason, that's won't work anymore.